homework7/tests/test_hw2.py

```diff
@@ -0,0 +1,15 @@
+from homework7.hw2 import backspace_compare
+
+
+def test_two_str_with_same_output_str():
+    """Test that function return True if we "#" symbol like backspace"""
+    str_1 = "we#c"
+    str_2 = "wetr###c"
+    assert backspace_compare(str_1, str_2)
+
+
+def test_two_str_with_not_the_same_output_str():
```

Comment:
add more test cases. what if strings start with #? what if one has # another not? etc.